app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -0,0 +1,29 @@
+function toggleModActionsMenu() {
+  document.querySelector('.mod-actions-menu').classList.toggle('showing');
+}
+
+export function initializeModActionsMenu() {
+  // eslint-disable-next-line no-undef
+  const user = userData();
+  const articlePath = `${
+    document.getElementById('article-show-container').dataset.path
+  }/actions_panel`;
+  const articleAuthorId = document.getElementById('article-show-container')
+    .dataset.authorId;
```

Comment:
You can use destructuring on the dataset.

```suggestion
  const { authorId: articleAuthorId, path } = document.getElementById('article-show-container').dataset;
  const articlePath = `${path}/actions_panel`;
```

Reply:
Wow TIL that you can destructure and rename the keys 😱